robot/control.py

```diff
@@ -0,0 +1,20 @@
+
+def check_for_obstacle(obstacle_sensor):
```

Comment:
Shouldn't this function return something? For example True when it detects something and False otherwise?